templates/not_logged.html

```diff
@@ -0,0 +1,13 @@
+{% extends 'global/base.html' %}
+
+{% block content %}
+    <div class="container">
+        <div class="row">
+            <div class="col-md-12">
+                <h2>This page is restricted to Django Girls workshops' organizers</h2>
+
+                <p>If you're an organizer, you need to <a href="https://djangogirls.org/admin">log in</a> to see this page.</p>
```

Comment:
If we manage to use `@login_required` then this template should not be necessary. But if that doesn't work, you can use `{% url 'admin:index' %}` instead of hardcoding the link.